contracts/Content/UniqueContentStorage.sol

```diff
@@ -108,8 +109,10 @@ contract UniqueContentStorage is IUniqueContentStorage, MultipleRoyalties {
         royaltyAmounts[0] = _originalRoyaltyAmount;
 
         (, uint256 _originalRoyaltyRate) = IERC2981Upgradeable(uniqueAssetInfo[_uniqueId].contentAddress).royaltyInfo(uniqueAssetInfo[_uniqueId].tokenId, 1e6);
+        // if it's greater than 20 percent ignore the remaining royalties
+        if (_originalRoyaltyRate >= 2e5) {}
```

Comment:
instead of if (originalRoyaltyRate >= 2e5) {} else {..}

just do:
if (originalRoyaltyRate <= 2e5) { ... stuff }

Reply:
Also, 20000 should be fine, so the oringal should have been if (originalRoyatlyRate > 2e5) { skip }

Reply:
In the original code, I was thinking that if the original royalty rate was 20000, it would have triggered the else statement which would split the remainder (which in that case would be zero) proportionally between the remaining addresses. Skipping the code entirely would achieve the same thing except it wouldn't register the receiver addresses, who wouldn't be paid anything anyway.

Reply:
Ah, you're right. That is correct.